js/src/forum/ForumApplication.js

```diff
@@ -94,6 +95,7 @@ export default class ForumApplication extends Application {
 
     this.pane = new Pane(document.getElementById('app'));
     this.composer = m.mount(document.getElementById('composer'), Composer.component());
+    this.cache.discussionList = new DiscussionListState();
```

Comment:
Perhaps this should go in a separate method? I don't feel like it belongs in `mount()`, that's for mounting stuff to the DOM.

Also - not entirely sure it belongs in the Application class at all; not sure where else it would go though.

Reply:
Good point, since refresh is no longer in the constructor, I moved it out of the method.

Unless we want to add another global variable, I'd say we can keep all this in app for now, and then change it around when we get to Part 2 of the state refactor (which would be after the TS/Mithril 2 upgrades)